src/garage/torch/policies/base.py

```diff
@@ -0,0 +1,53 @@
+
```

Comment:
Actually, this class should probably go in src/garage/np/policies/base.py. Perhaps we can put it there, and call it `PolicyBase`. @zhanpenghe, what do you think?

We're fairly close to deleting `Parameterized`, so the old `Policy` class shouldn't really be needed. And Tensorflow specific stuff in that class (e.g. the `_variable_scope`) should probably be moved to the `tf` branch.

Reply:
Do I need to move and rename the policy class?
@krzentner @zhanpenghe

Reply:
As far as I can tell, the policy class in [src/garage/np/policies/base.py](https://github.com/rlworkgroup/garage/blob/master/src/garage/np/policies/base.py) is only ever used in a few tests, and doesn't use `Parameterized` at all. We can move it in a later PR though, in case doing so causes other breakages.

Reply:
i agree -- please just correct the one in np.

Reply:
I agree too.

Reply:
Is it ok to do this job on new PR?
I am worried that this change could affect other codes

Reply:
Sure, implementing this change in a later PR sounds fine.